### DISCUSSION TITLE
More Outline effects

### DISCUSSION THREAD

**Discussion by Superneutrino** [SilenceLeo/snowb-bmf#72] [Ideas]
Hi. Some bitmap font editors like Hiero and others have option to add almost unlimited number of outlines to font. This would work really nice with creative fonts with multiple color strokes.  I tried to add additional stroke in Photoshop, but SnowB is  smart, and ignores the transparent stroke I gave to it.

Just a thought.

**Comment by SilenceLeo** [comment]
Hi! Thank you for using SnowB and reaching out with your question. 

SnowB ignores the transparent boundaries when exporting images in order to optimize the file size, but it does not affect the result of the Bitmap Font. The description file contains xoffset, yoffset, width, and height parameters to determine the empty area. You can use graphic editing software like Photoshop to create more complex glyphs and import them into SnowB as images.
